src/System.Windows.Forms/src/System/Windows/Forms/ListViewGroup.ListViewGroupAccessibleObject.cs

```diff
@@ -13,6 +15,7 @@ public partial class ListViewGroup
         internal class ListViewGroupAccessibleObject : AccessibleObject
         {
             private readonly ListView _owningListView;
+            private readonly ListViewAccessibleObject _internalParent;
```

Comment:
What's an "ListViewGroupAccessibleObject  internal parent"?
```suggestion
            private readonly ListViewAccessibleObject _owningListViewAccessibilityObject;
```